### DISCUSSION TITLE
Get a list of the game scores for players

### DISCUSSION THREAD

**Discussion by pbcurran** [cwendt94/espn-api#291] [ESPN API]
Hi I'm wondering if there is way to get something like a list of the number of NBA fantasy points  scored by a player for each game of the season. Something like the below I mean

scores = {
                  gameOne: 50.0,
                  gameTwo: 45.0,
                  ....
}

I've found the weekly total player scores in the box_scores class so maybe even even the number of games played by the player in the box score period would also be great

Really the work put into this, thanks!

**Comment by cwendt94** [comment]
HI @pbcurran currently NBA games are broken down by `scoring_period` which can be passed in to the `box_score` call. However its not easy to know which `scoring_periods` a player is playing. I plan on looking into adding the number of games a player played for that matchup and also adding a `player_info` function that will return all of the detailed stats for a player

**Reply by pbcurran** [reply]
ok cool I appreciate it. I'll have a look into the scoring_period call as well. Thanks again

**Comment by shibedroid** [comment]
Hi, @cwendt94. Did you find a way to determine how many games a player has played?

**Reply by cwendt94** [reply]
The player class now has a schedule field which holds the team and date they played. It looks like this where the keys are the scoring period.
```
{'2': {'team': 'BKN', 'date': datetime.datetime(2018, 10, 17, 23, 0)}}
```
You can also use the [player_info](https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball) functionality which will return the player class and fill out all of their stats for the entire season and every game they played.

Is this what you are looking for?

**Reply by shibedroid** [reply]
not exactly, what I did is that I modified the X-filter for the kona_player_info so I could get the historical data from the past weeks of the current season and I noticed that if a player had not played a certain week, the 'stats' dict in 'player' was an empty dictionary, but if the player played but got zero points, the dict was not empty, so I checked how many empty weeks a player had and I substracted it from the number of weeks elapsed since kickoff and voila.

thank you anyway!! excelent job, I use your API as an inspiration all the time
